sdks/python/apache_beam/transforms/core.py

```diff
@@ -1427,6 +1478,16 @@ def __init__(self, fn, *args, **kwargs):
     if not isinstance(self.fn, DoFn):
       raise TypeError('ParDo must be called with a DoFn instance.')
 
+    # DoFn.process cannot allow both return and yield
+    if _check_fn_use_yield_and_return(self.fn.process):
+      _LOGGER.warning(
+          'The yield and return statements in the process method '
```

Comment:
I would rephrase the beginning as: `Using yield and return in the process method can lead to unexpected behavior, see: https://github.com/apache/beam/issues/22969.`

Reply:
Sounds good. Done.